doc/hands_on/hands_on_dynamodb_vs.md

```diff
@@ -0,0 +1,229 @@
+# Hands on DynamoDB Virtual Schema
+
+In this Hands on guide we show you how to explore the new Virtual Schema for DynamoDB.
+
+We will create a mapping from DynamoDB's semi structured documents to a relational exasol table.
+
+
+## DynamoDB Setup
+
+First we need a DynamoDB.
+For that there are two options:
+
+* DynamoDB on AWS
+  * free in [AWS Free Tier](https://aws.amazon.com/de/free/) (credit card required)
+* Local test DynamoDB
+  * free
+  * for testing only
+
+You can use both options to follow this guide. 
+
+### DynamoDB on AWS
```

Comment:
This part looks as GitHub documentation. Just try to add more words

For example:
`If you decided to use DynamoDB on AWS, let us show you how to set it up.`

And so on...